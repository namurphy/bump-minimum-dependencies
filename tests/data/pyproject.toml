[project]
name = "temporary-package"
version = "0.1.0"

requires-python = ">=3.14"
dependencies = [
    "astropy>=4",
    "plasmapy>=0.3.1",
    "numpy>=1.22.4",
]

[build-system]
requires = ["uv_build>=0.9.7,<0.10.0"]
build-backend = "uv_build"

# "cabinet>=2023.4.30.1,!=2024.7.17.1,!=1!2.1.1"
# "requests>=2.19.1"