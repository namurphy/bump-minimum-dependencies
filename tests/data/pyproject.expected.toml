[project]
name = "temporary-package"
version = "0.1.0"

requires-python = ">=3.14"
dependencies = [
    "astropy>=6",
    "plasmapy>=2024.2",
    "numpy>=2",
]

[build-system]
requires = ["uv_build>=0.9.7,<0.10.0"]
build-backend = "uv_build"
